### PR TITLE
Feat : Added homePage

### PR DESCRIPTION
**Pull request by RahulSingh9131**
Home page
ShoestoreLandingPage

### CODE REVIEW

File: src/css/main.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @import url("./reset.css");
 @import url("./utils.css");
 @import url("./variables.css");
+@import url("./screens/home.css");
 
 @import url("./mediaQueries.css");
```
Comment (rahulcs754) [inline src/css/main.css:6]: I think you should have add only your single component lib link

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                                <h1>Sports Shoes</h1>
+                                <p>Shop from a wide range of sports shoes.</p>
+                                <Link to="/" className='btn'>see More</Link>
+                            </div>
+                        </div>
+                        <div className="box">
+                            <img src="https://images.unsplash.com/photo-1481729379561-01e43a3e1ed4?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=970&q=80" alt="category-image"/>
+                            <div className="content">
+                                <h1>Boots</h1>
+                                <p>Shop from a wide range of boots.</p>
```
Comment (raman1999) [inline src/pages/Home.js]: u  can use api/categories of mockbee to get your categories data from there and can map it directly

**Review by raman1999 — COMMENTED** [review]
suggested minor changes,rest looks good

**Review by rahulcs754 — APPROVED** [review]
After checking your code 
I think you should have used the CSS module for your page.
Using the CSS module, you can provide separate CSS code for a particular component.
You can use the CDN link for your images that will access fast.

File: src/css/main.css (hunk 1 above)
Comment (RahulSingh9131) [reply src/css/main.css:6]: yes i have added that in App.css file.

**Pull request merged**